src/Mgate/TresoBundle/Resources/views/BV/voir.html.twig

```diff
@@ -67,21 +67,21 @@
                 <td>{{ ((cotisation.surBaseURSSAF ? bv.assietteDesCotisations : bv.remunerationBrute) * cotisation.tauxPartEtu) | money(false) }}</td>
             </tr>
         {% endfor %}
-        <tr class="active money">
+        <tr class="active">
             <th style="text-align: left;">{{ 'treso.total_cotisations_dues'|trans({}, 'treso') }}<br>(indexées sur l'assiette de cotisation)</th>
-            <th>{{ bv.assietteDesCotisations | money }}&nbsp;€</th>
-            <th>{{ (bv.tauxPartJunior['baseURSSAF'] * 100) | money }}&nbsp;%</th>
-            <th>{{ bv.partJunior(true)['baseURSSAF'] | money }}&nbsp;€</th>
-            <th>{{ (bv.tauxPartEtu['baseURSSAF'] * 100) | money }}&nbsp;%</th>
-            <th>{{ bv.partEtudiant(true)['baseURSSAF'] | money }}&nbsp;€</th>
+            <th class="money">{{ bv.assietteDesCotisations | money }}&nbsp;€</th>
```

Comment:
Because of that change, columns' content of columns is not aligned with headers anymore. It makes the table very difficult to read.